app/jobs/cache_browser_channels_json_job_v2.rb

```diff
@@ -17,6 +20,7 @@ def perform
     end
 
     if result
+      Rails.cache.write(LAST_WRITTEN_AT_KEY, Time.now)
```

Comment:
Will this key / value get overwritten by the other jobs calling it? Maybe a unique key for each job?

Reply:
Single worker, so it's fine